src/lib/Theme.ts

```diff
@@ -319,15 +320,19 @@ export default class Theme {
                 }
             });
             Logger.success('Theme syncing DONE...');
+            let Url = `https://${AVAILABLE_ENVS[currentContext.env]}`;
```

Comment:
change variable name, the variable name should start with small case only @anuragpandey1115